addons/nametags/stringtable.xml

```diff
@@ -223,6 +223,12 @@
             <Chinese>如果這個設定啓用，系統會使用BIS_fnc_sideIsFriendly來代替陣營檢查</Chinese>
             <German>Wenn diese Einstellung aktiviert ist, verwendet das System BIS_fnc_sideIsFriendly, anstatt nur die Seite</German>
         </Key>
+        <Key ID="STR_dui_nametags_useLIS">
+            <English>Visualize Vehicle Crew</English>
+        </Key>
+        <Key ID="STR_dui_nametags_useLIS_desc">
+            <English>Detect Crew Inside Vehicles</English>
```

Comment:
I think it should be made clear to the end user that this will run additional checks.

Reply:
if you give me a better wording I can add it 😄

Reply:
```suggestion
            <English>Detect Crew Inside Vehicles. Runs extra checks and routines when enabled.</English>
```